catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveCommand.java

```diff
@@ -118,7 +118,8 @@ private Object executeRemoveFromStore() throws Exception {
             return null;
         }
 
-        DeleteRequestImpl request = new DeleteRequestImpl(ids.toArray(new String[numberOfMetacardsToRemove]));
+        DeleteRequestImpl request =
+                new DeleteRequestImpl(ids.toArray(new String[numberOfMetacardsToRemove]));
```

Comment:
Might consider breaking this into more steps/utility method to preserve readability.